### DISCUSSION TITLE
Instructivo para configurar VS Code

### DISCUSSION THREAD

**Discussion by JMGamero** [Launch-X-Latam/MisionFrontEnd#139] [Q&A]
Estimados, buenas tardes. Seria posible que nos guiaran en la configuracion del VS Code? Ya sea, los plug ings recomendables y como instalar los lenguajes y/o librerias que estaremos utilizando a lo largo del curso.

Desde ya,  muchas gracias y saludos!

Juan Manuel.

**Comment by doguedogue** [comment]
Que tal, hay muchos cursos para VS Code en la red, realmente tu vas agregando las extensiones que te hacen más fácil o te ayudan.
La idea es que las extensiones te faciliten tareas que ya conoces y no que hagan todo por ti, o luego no sabras como se hizo (ja)

Si vas a empezar con html hay varias para formatear el código (Prettier)
Hay también para javascript (Beutify), vue, angular, incluso para python, java, php, c# etc.

A mi en lo personal me gustan:  
- Beutify, 
- Prettier, 
- Bracket Pair Colorizer 2,  
- angular, 
- boomarks, 
- Live Server, 
- Material Icon Theme, 
- Activitus Bar
- etc.

Siempre hay como quitarlas si no te gustan o reinstalar (ja) Aprende a usar VS Code como experto, mi uso favorito es editar varias líneas a la vez (https://stackoverflow.com/questions/30037808/multiline-editing-in-visual-studio-code)
